cypress/integration/search-highlighting.ts

```diff
@@ -0,0 +1,7 @@
+describe('search highlighting', () => {
+  it('text searched is highlighted', () => {
+    cy.visit('examples/search-highlighting')
+    cy.get('input[type="search"]').type('text')
+    cy.get('[data-cy="search-highlighted"]').should('have.length', 2)
```

Comment:
Put these strings in a variable to make it more obvious to readers what the test is doing. Something like...

```ts
const searchInputField = 'input[type="search"]'
const highlightedText = `[data-cy="search-highlighted"]`
```